homeassistant/components/laundrify/strings.json

```diff
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "step": {
+      "init": {
+        "description": "Please enter your personal Auth Code that is shown in the laundrify-App.",
+        "data": {
+          "code": "Auth Code (xxx-xxx)",
+          "poll_interval": "Poll Interval (in seconds)"
```

Comment:
can be removed

Reply:
Removed in 95018cedae969e535cba4f82c75d63664d162291